packages/web-twig/src/Resources/components/Collapse/Collapse.twig

```diff
@@ -2,8 +2,8 @@
 {%- set props = props | default([]) -%}
 {%- set _class = props.class | default(null) -%}
 {%- set _id = props.id -%}
-{%- set _breakpoint = props.breakpoint -%}
-{%- set _parent = props.parent -%}
+{%- set _breakpoint = props.breakpoint | default(null) -%}
```

Comment:
Please change the readme file as well. (just fill default value in the API table)